sdk/tables/azure-data-tables/test/ut/table_client_test.hpp

```diff
@@ -0,0 +1,60 @@
+#pragma once
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "test/ut/test_base.hpp"
+
+#include <azure/data/tables/models.hpp>
+#include <azure/data/tables/rest_client.hpp>
+#include <azure/data/tables/transactions.hpp>
+
+namespace Azure { namespace Data { namespace Test {
+  enum class AuthType
+  {
+    Key = 0x0,
+    SAS = 0x1,
+    ConnectionString = 0x2,
+  };
+
+  class TablesClientTest : public Azure::Storage::Test::StorageTest,
+                           public ::testing::WithParamInterface<AuthType> {
+  protected:
+    void SetUp() override;
+    void TearDown() override;
+    Azure::Data::Tables::TableClient CreateTableClientForTest(
+        Tables::TableClientOptions& clientOptions);
+
+    Azure::Data::Tables::TableClient CreateKeyTableClientForTest(
+        Tables::TableClientOptions& clientOptions);
+    std::string GetConnectionString()
+    {
+      const static std::string ConnectionString = "";
+
+      if (!ConnectionString.empty())
+      {
+        return ConnectionString;
+      }
+      const static std::string envConnectionString = GetEnv("STANDARD_STORAGE_CONNECTION_STRING");
```

Comment:
Same comment

Reply:
Yes i connect to the storage accoutn, it's the same conn string that could be used by blob service for example